### ISSUE TITLE
SlevomatCodingStandard.Functions.UselessParameterDefaultValue does not understand variadic parameters

### ISSUE THREAD

**Issue by dakorpar** [slevomat/coding-standard#739]
for example 

```php
...
public function __construct(string $a = 'something', string $b = 'b', ...$parameters); 
//this triggers error and it will be changed to
```
```php
public function __construct(string $a, string $b, ...$parameters); 
// this is never wanted.
```

**Comment by kukulich** [comment]
https://github.com/slevomat/coding-standard/pull/740#issuecomment-547326884
